src/main/java/ledger/user_interface/ui_controllers/AccountPopupController.java

```diff
@@ -61,6 +61,7 @@ public void initialize(URL fxmlFileLocation, ResourceBundle resources) {
                 task.startTask();
             } catch (Exception e) {
                 System.out.println("Error on account submission: " + e);
+                this.setupErrorPopup("Error on account submission - please check that all fields have valid values.");
```

Comment:
This catch does not catch error on submission but rather error on getting account or balance. Both of those methods should catch their own errors also